_posts/2024-04-05-Configurable-Automation-for-OpenSearch-ML-Use-Cases.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Configurable automation for OpenSearch ML use cases"
+authors:
+    - kazabdu
+    - amitgalitz
+    - dwiddis
+    - jpalis
+    - hnyng
+    - ohltyler
+    - minalsha
+date: 2024-04-05
+categories:
+  - technical-posts
+meta_keywords: Flow Framework, OpenSearch plugins, Machine Learning
+meta_description: Explore the simplicity of integrating Machine Learning capabilities within OpenSearch through an innovative and groundbreaking framework designed to simplify complex setup tasks.
+---
+
+In OpenSearch, to use machine learning (ML) offerings, such as semantic, hybrid, and multimodal search, you often have to grapple with complex setup and preprocessing tasks. Additionally, you must write verbose queries, which can be a time-consuming and error-prone process.
+
+In this blog post, we introduce the OpenSearch Flow Framework plugin, [released in version 2.13](https://opensearch.org/blog/2.13-is-ready-for-download/) and designed to streamline this cumbersome process. By using this plugin, you can simplify complex setups with just one click. We've provided automated templates, enabling you to create connectors, register models, deploy them, and register agents and tools through a single API call. This eliminates the complexity of calling multiple APIs and orchestrating setups based on the responses.
```

Comment:
Nit: "click" is kind of GUI centric and we're still API.  Can we maybe say "one simple API call?"